utils_test.py

```diff
@@ -327,3 +327,13 @@ def test_get_supported_audio_language_description_with_invalid_code(self):
             Exception, 'Unsupported audio language code: invalid_code'):
             utils.get_supported_audio_language_description(
                 invalid_language_code)
+
+    def test_snake_case_to_camel_case(self):
+        snake_case_str1 = utils.snake_case_to_camel_case('user_id_number')
```

Comment:
The variable name is confusing because the variable is actually storing something that is in camel case.

Reply:
Done.